api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -1427,4 +1428,47 @@ public static void setUseSystemClassLoader(boolean useSystemClassLoader) {
 	public static boolean isUseSystemClassLoader() {
 		return getServiceContext().isUseSystemClassLoader();
 	}
+	
+	/**
+	 * Evicts the entity data for a particular entity "instance".
+	 *
+	 * @param sessionFactory
+	 * @since 2.6.0
+	 */
+	public static void evictSingleEntity(SessionFactory sessionFactory, Class<?> entityClass, String uuid) throws IllegalArgumentException {
+		if(StringUtils.isBlank(uuid)){
+			throw new IllegalArgumentException();
+		}
+		log.debug("Clearing DB cache for entity: {} with uuid: {}", entityClass, uuid);
+		OpenmrsObject object = (OpenmrsObject) ((BaseDelegatingResource) resource).getByUniqueId(uuid);
+		sessionFactory.getCache().evictEntity(entityClass, object.getId());
+		sessionFactory.getCache().evictCollectionRegions();
+		sessionFactory.getCache().evictQueryRegions();
+	}
+
+	/**
+	 * Evicts all entity data from the given region (i.e. for all entities of type).
+	 *
+	 * @param sessionFactory
+	 * @since 2.6.0
+	 */
+	public static void evictAllEntities(SessionFactory sessionFactory, Class<?> entityClass) throw IllegalArgumentException{
+		if(entityClass == null)
```

Comment:
Are you not supposed to include curly braces here. Is it possible to include this on one line, Perhaps it can look more appropriate.